src/Auth0.AuthenticationApi/AuthenticationApiClient.cs

```diff
@@ -350,7 +351,34 @@ await AssertIdTokenValidIfExisting(response.IdToken, request.ClientId, request.S
         }
 
         /// <inheritdoc/>
+        public async Task<MfaOobTokenResponse> GetTokenAsync(MfaOobTokenRequest request, CancellationToken cancellationToken = default)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            var body = new Dictionary<string, string>()
+            {
+                { "grant_type", "http://auth0.com/oauth/grant-type/mfa-oob" },
+                { "client_id", request.ClientId },
+                { "client_secret", request.ClientSecret },
```

Comment:
This is added through `ApplyClientAuthentication`
```suggestion
```

Reply:
made change